domain/src/main/java/com/epam/esm/domain/dto/TagDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.domain.dto;
+
+import com.epam.esm.domain.entity.Certificate;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class TagDTO {
+
+    private Long id;
+    private String name;
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private List<Certificate> certificates;

```

Comment:
Could you explain why TagDTO has certificates as part of itself?

Reply:
I've read about many-to-many relationship, so there was a recommendation to keep a list of objects, to show the relation. but now, after i've wrote this explanation, i understood that there's no need, cause there's no use of this list. thank's